resources/css/filament/company/invoiceplane-blue.css

```
@@ -1,8 +1,10 @@
 @import 'tailwindcss';
 @import '../../../../vendor/filament/filament/resources/css/theme.css';
+@import '../../../../vendor/awcodes/mason/resources/css/plugin.css';
 
 @source '../../../../Modules/**/resources/views/**/*';
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)